examples/react-ts/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
```

Comment:
If we are going to include tests it should be `bun:test` in the bun example :)

Reply:
I don't know how to fix that. all i wanted to do is add the react typescript template. If anyone can help with this would be awesome.